transcom/mymove
# Int b 21901 add destination too col to moves table

## [B-21901](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1038020&RoomContext=TeamRoom%3A1007470&concept=TeamRoom)

## Summary
This PR adds the too_destination column to the moves table to track the assigned office_user for the destination glboc queue.
Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Run the migration `make db_dev_reset`  then `make db_dev_migrate`
3. Verify that the moves table has the new column `too_destination_assigned_id` added. 
     - I verified with DBeaver by just running (`SELECT * FROM moves;`) then make sure the new column is there.


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![image](https://github.com/user-attachments/assets/7db28277-c1eb-4460-b3fa-3098e6615b3b)
